src/Searchable.php

```diff
@@ -21,6 +21,22 @@ trait Searchable
      */
     private static $filterCache = [];
 
+    /**
+     * Classes using this trait use $searchable to define searchable field
+     *
+     * @var array $searchable
+     */
+    // public $searchable = [];
```

Comment:
Should this be commented out? Probs better keeping it in as a default

Reply:
Yeah because it's in a trait you can't override properties